src/table/body-cell/index.tsx

```diff
@@ -58,6 +62,8 @@ function TableCellEditable<ItemType>({
         isVisualRefresh && styles['is-visual-refresh']
       )}
       onClick={!isEditing ? onEditStart : undefined}
+      onMouseEnter={() => setHasHover(true)}
```

Comment:
[minor, possibly premature over-optimization] is there a performance impact of these not using `useCallback` and therefore creating new instances on each render?